src/ui/Editor.js

```diff
@@ -205,7 +205,9 @@ class Editor extends React.PureComponent<any, any, any> {
   }
 
   componentWillUnmount(): void {
+    this._autoFocusTimer && clearTimeout(this._autoFocusTimer);
     this._editorView && this._editorView.destroy();
+    this._editorView && (this._editorView._props = {});
```

Comment:
I didn't put this in the `destroy` in `prosemirror-view` for now since we still have our own code instances sticking around in memory that may be contributing to the problem.

Reply:
I see.

So you should do this at `src/ui/CustomEditorView.js`

```
class CustomEditorView {
  destroy() {
    super();
    this._props = {};
  }
}
```